james/types.go

```diff
@@ -355,57 +356,53 @@ func (j *JMAPClient) RenewSession() error {
 	return <-errChan
 }
 
-func (j *JMAPClient) renew() error {
-	var lastErr error
-	err := wait.PollUntilContextTimeout(
+func (j *JMAPClient) renew() (err error) {
+	_ = wait.PollUntilContextTimeout(
```

Comment:
No need to declare err in the return params. Just use the err returned from the wait.PollUntilContextTimeout() and return. Sometimes it confusing to track the err